app/main.py

```diff
@@ -1,8 +1,45 @@
+from __future__ import annotations
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
```

Comment:
all arguments from a newline, including the first one after the opening parenthesis, and the closing parenthesis as well